scenarios/command/c/command_client/main.c

```diff
@@ -173,6 +202,25 @@ int main(int argc, char* argv[])
       {
```

Comment:
A comment on the previous if statement, should there be a macro to replace the 2 to be consistent with the wait on line 189 which uses `COMMAND_TIMEOUT_SEC`? Maybe something like `COMMAND_RETRY_SEC`?